app/routes/docs/GamePopup.tsx

```diff
@@ -0,0 +1,88 @@
+import { useState, useEffect } from "react";
+import { useGameStore } from "~/lib/zustand/game";
+
+const GamePopup = () => {
+    const [isVisible, setIsVisible] = useState(true);
+    const [typedText, setTypedText] = useState("");
+
+    const currentLevel = useGameStore((state) => state.currentLevel);
+
+    const levelDescriptions: Record<string, string> = {
+        playground: "In the playground, you have access to all features and can freely test everything.",
+        "1.1": "Your goal is to create a character controller for the raccoon and navigate it to the goal flag.",
+    };
+
+    const description = levelDescriptions[currentLevel] || "No description available for this level.";
+
+    // Typing effect
+    useEffect(() => {
+        let index = 0;
+
+        const interval = setInterval(() => {
+            if (index <= description.length) {
+                setTypedText(description.slice(0, index));
+                index++;
+            } else {
+                clearInterval(interval);
+            }
+        }, 10);
+
+        return () => {
+            clearInterval(interval);
+            setTypedText("");
+        };
+    }, [description]);
+
+    if (!isVisible) {
+        return null;
+    }
+
+    // Game popup
+    return (
+        <div className="fixed inset-0 flex items-center justify-center z-50">
+            <div className="flex flex-col w-[50%] max-h-[80vh] rounded-lg border-3 p-5 font-mono text-white bg-gray-800/95 border-blue-300">
```

Comment:
```suggestion
            <div className="flex flex-col w-[50%] max-h-[80vh] rounded-lg border-3 p-5 font-mono text-white bg-slate-800/95 border-blue-300">
```

we've been using slate as a neutral color, maybe use this here too?

Reply:
Done.